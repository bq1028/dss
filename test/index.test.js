const dss = require('../')

describe('dss', () => {
  it('compiles', async () => {
    const styles = await dss(`
      .a { color: red; }
      .a:hover { color: blue; }
      @media screen and (min-width: 30px) {
        .a { color: hotpink }
      }
    `)

    expect(styles).toMatchSnapshot()
    expect(dss.css()).toMatchSnapshot()
  })
})
