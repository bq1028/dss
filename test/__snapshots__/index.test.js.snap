// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dss compiles 1`] = `
Object {
  "a": Array [
    "dss_rfc3hq-169mlyl",
    "dss_16bwgo5-11z5xnj",
    "dss_1u3exn4-nfznl2",
  ],
}
`;

exports[`dss compiles 2`] = `".dss_1u3exn4-nfznl2:hover{color:blue}.dss_rfc3hq-169mlyl{color:red}@media screen and (min-width: 30px){.dss_16bwgo5-11z5xnj{color:hotpink}}"`;
